docs: add note that CLI version 15 doesn't create karma.conf.js in new projects (#48590)
angular/angular: aio/content/guide/testing-code-coverage.md

# Find out how much code you're testing

The Angular CLI can run unit tests and create code coverage reports.
Code coverage reports show you any parts of your code base that might not be properly tested by your unit tests.

<div class="alert is-helpful">

> If you'd like to experiment with the app that this guide describes, <live-example name="testing" noDownload>run it in your browser</live-example> or <live-example name="testing" downloadOnly>download and run it locally</live-example>.

Reviewer: Why this change? Application should be preferred over app.

</div>

> To create a coverage report run the following command in the root of your project.

Reviewer: ```suggestion
To generate a coverage report run the following command in the root of your project.
```

## Code coverage enforcement

The code coverage percentages let you estimate how much of your code is tested.

> If your team decides on a set least amount to be unit tested, enforce this amount with the Angular CLI.

Reviewer: ```suggestion
If your team decides on a set a minimum amount to be unit tested, enforce this minimum with the Angular CLI.
```

> If your team decides on a set smallest amount to be unit tested, enforce this amount with the Angular CLI.

Author: Sorry, I only saw this comment now, and I ended up using "least". Should I also change all the instances where "minimum" was used back to its original state, or is it just the instance I ended up using "smallest"?

> If your team decides on a set smallest amount to be unit tested, enforce this amount with the Angular CLI.

Reviewer: Suggested the changes.

> For example, suppose you want the code base to have at least 80% code coverage.

Reviewer: ```suggestion
For example, suppose you want the code base to have a minimum of 80% code coverage.
```

</code-example>

> The `check` property causes the tool to enforce at least 80% code coverage when the unit tests are run in the project.

Reviewer: ```suggestion
The `check` property causes the tool to enforce at a minimum of 80% code coverage when the unit tests are run in the project.
```
